api/src/main/java/org/pmiops/workbench/cdr/ConceptBigQueryService.java

```diff
@@ -98,21 +94,4 @@ private void generateParentChildLookupSql(
       sqlBuilder.append(" unnest(@" + conceptIdsParam + ")");
     }
   }
-
```

Comment:
This method was only used by the DataSetServiceImpl, so instead of injecting this service it makes sense to move it to the DataSetServiceImpl

Reply:
agreed!